### ISSUE TITLE
Installation for India Compliance failed due to an error

### ISSUE THREAD

**Issue by mumbaigirl** [resilient-tech/india-compliance#1631]
### Information about bug

Installing india_compliance...
Updating DocTypes for india_compliance: [========================================] 100%
Setting up Income Tax...
Setting up GST...
Patching Existing Data...
Installation for India Compliance failed due to an error. Please try re-installing the app or report the issue on https://github.com/resilient-tech/india-compliance/issues/new if not resolved.
An error occurred while installing india_compliance: (1054, "Unknown column 'tabPurchase Invoice Item.is_non_gst' in 'where clause'")
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 462, in install_app
    _install_app(app, verbose=context.verbose, force=force)

### Version

ERPNext: v15.10.6 (version-15)
Frappe Framework: v15.11.0 (version-15)
India Complaince version-15

It was working smoothly, however after recent ERPNext update, it has started throwing this error.

System
4GB RAM, Ubuntu 22.04
50 GB SSD

### Relevant log output / Stack trace / Full Error Message.

```shell
App erpnext already installed

Installing india_compliance...
Updating DocTypes for india_compliance: [========================================] 100%
Setting up Income Tax...
Setting up GST...
Patching Existing Data...
Installation for India Compliance failed due to an error. Please try re-installing the app or report the issue on https://github.com/resilient-tech/india-compliance/issues/new if not resolved.
An error occurred while installing india_compliance: (1054, "Unknown column 'tabPurchase Invoice Item.is_non_gst' in 'where clause'")
Traceback with variables (most recent call last):
  File "apps/frappe/frappe/commands/site.py", line 462, in install_app
    _install_app(app, verbose=context.verbose, force=force)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mohsinalimat** [comment]
Same issue with me.

**Comment by ljain112** [comment]
Sorry for inconvenience caused.
We have already fixed this issue here #1620 .
We will inform you one's released.

**Comment by sagarvora** [comment]
This fix is released. You may update your app and try again.
